docs/reference/commandline/stats.md

```diff
@@ -11,13 +11,22 @@ weight=1
 
 # stats
 
-    Usage: docker stats CONTAINER [CONTAINER...]
+    Usage: docker stats [OPTIONS] [CONTAINER...]
 
     Display a live stream of one or more containers' resource usage statistics
 
+      --all=false        Show all containers (default shows just running)
       --help=false       Print usage
       --no-stream=false  Disable streaming stats and only pull the first result
 
+Running `docker stats` on all running containers
+
+    $ docker stats
+    CONTAINER           CPU %               MEM USAGE / LIMIT     MEM %               NET I/O             BLOCK I/O
```

Comment:
For readability, I sometimes reduce the spacing between columns in the output. Just a suggestion, feel free to ignore :-)